doc/rst/users/build.rst

```diff
@@ -6,144 +6,198 @@ Build SCR
 Dependencies
 ------------
 
-SCR has several dependencies. A C compiler, MPI, CMake, and pdsh are
-required, as are several component libraries from the ECP VeloC project. The others are optional, and when they are
-not available some features of SCR may not be available.
+SCR has several required dependencies.
+Others are optional, and when they are not available
+some features of SCR may not be available.
 
 Required:
 
 * CMake, Version 2.8+
-* Compiler (C, C++, and Fortran)
-* MPI
-* pdsh (https://github.com/grondo/pdsh)
+* Compilers (C, C++, and Fortran)
+* MPI 3.0+
+* pdsh (https://github.com/chaos/pdsh)
+* DTCMP (https://github.com/llnl/dtcmp)
+* LWGRP (https://github.com/llnl/lwgrp)
 * AXL (https://github.com/ECP-VeloC/AXL)
-* KVTree (https://github.com/ECP-VeloC/KVTree)
 * er (https://github.com/ECP-VeloC/er)
-* redset (https://github.com/ECP-VeloC/redset)
+* KVTree (https://github.com/ECP-VeloC/KVTree)
 * rankstr (https://github.com/ECP-VeloC/rankstr)
+* redset (https://github.com/ECP-VeloC/redset)
 * shuffile (https://github.com/ECP-VeloC/shuffile)
 * spath (https://github.com/ECP-VeloC/spath)
-* DTCMP (https://github.com/llnl/dtcmp)
-* LWGRP (https://github.com/LLNL/lwgrp)
 
 Optional:
 
-* libyogrt (for determining length of time left in the current allocation) (https://github.com/llnl/libyogrt)
+* libyogrt (for time remaining in a job allocation) (https://github.com/llnl/libyogrt)
 * MySQL (for logging SCR activities)
 
-On Fedora, you can install the required dependencies with::
+To simplify the install process,
+one can use Spack or use the SCR :code:`bootstrap.sh` script and CMake.
 
-  sudo dnf groupinstall "Development Tools"
-  sudo dnf install cmake gcc-c++ mpi mpi-devel environment-modules zlib-devel pdsh
-  [restart shell]
-  module load mpi
```

Comment:
Please keep this in!  I sometimes build SCR on a bleeding-edge Fedora VM to catch build errors, and this is really useful.